onchain/src/art_peace.cairo

```diff
@@ -496,6 +496,11 @@ pub mod ArtPeace {
             self.factions_count.read()
         }
 
+        fn get_has_joined_faction(self: @ContractState) -> u32 {
```

Comment:
Calling this `get_user_memberships_count` and passing in the user address would be more descriptive.